Library/Formula/curl.rb

```diff
@@ -81,7 +82,7 @@ def install
     args << (build.with?("libssh2") ? "--with-libssh2" : "--without-libssh2")
     args << (build.with?("libidn") ? "--with-libidn" : "--without-libidn")
     args << (build.with?("libmetalink") ? "--with-libmetalink" : "--without-libmetalink")
-    args << (build.with?("gssapi") ? "--with-gssapi" : "--without-gssapi")
+    args << (build.with?("gssapi") ? "--with-gssapi" + (OS.mac? ? "" : "=#{Formula["homebrew/dupes/krb5"].opt_prefix}") : "--without-gssapi")
```

Comment:
Yes, once dependency on `krb5` is added change on line 85 is not necessary. My concern is that `curl` still builds even if it does not find `gssapi` when `--with-gssapi` is provided.